src/libs/clientHelpers/createClient.ts

```diff
@@ -41,6 +43,7 @@ export const createClient = async (instance: string): Promise<CozyClient> => {
   const stackClient = client.getStackClient()
   stackClient.setUri(instance)
   await stackClient.register(instance)
+  await changeLanguage(client.getInstanceOptions().locale)
```

Comment:
changeLanguage() called here

Reply:
> Is `client.getInstanceOptions()` supposed to work when not logged yet?

it doesn't work indeed. I'm not sure where to put it, there are a lot of options. Can't we use a single call for both login and creation?

Reply:
getClient, finalizeClient, check if lang can be used before login, add a fallback in nav to check if lang is correct

Reply:
setLang to phone lang on logout

Reply:
set phone lang on welcome screen